docs/topics/development/remote_settings.md

```diff
@@ -0,0 +1,116 @@
+# Remote Settings
+
+This page explains how to set up [Remote Settings][] locally, which can be
+useful when working on the [Blocklist feature](../blocklist.md). AMO must be
+running locally first. Make sure it's available at <http://olympia.test/>.
+
+## Configure addons-server
+
+Add the following configuration variables to the `local_settings.py` file:
+
+```
+# When using Docker Desktop - `host.docker.internal` is a special host to allow
+# containers to get access to the host system, but that won't work on Linux.
+REMOTE_SETTINGS_API_URL = "http://host.docker.internal:8888/v1/"
+REMOTE_SETTINGS_WRITER_URL = "http://host.docker.internal:8888/v1/"
+
+# For Linux, you need to find the IP address of the Remote Settings container:
+# REMOTE_SETTINGS_API_URL = "http://172.17.0.1:8888/v1/"
+# REMOTE_SETTINGS_WRITER_URL = "http://172.17.0.1:8888/v1/"
+```
+
+Next, reload everything by running `make up`.
+
+At this point, AMO should be able to find the Remote Settings local server that
+we're going to set up next.
+
+## Set up Remote Settings
+
+In order to set up Remote Settings, follow these steps:
+
+1. Clone <https://github.com/mozilla/remote-settings>
+2. Run `make start`
```

Comment:
I guess this is outside the docker `web` container and in a random place, or ... is it inside the `web` container?

Reply:
it's outside yeah